File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    
    //Ensures that user inputs a valid height
    while (height < 1 || height > 8) 
    {
        height = get_int("Enter a height from 1 to 8 inclusive\n");
    }
    
    //Iterated for loop in order to sequentially print the hashes in a pyramid fashion
    for (int i = 0; i < height; i++)
    { 
        //Enters spaces so that pyramid is right-aligned
        for(int k = 0; k < height - i - 1; k++)
        {
            printf(" ");
        }
        
        for(int j = 0; j <= i; j++)
        {
            printf("#");  
        }
        
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

# Collects valid user input for height of pyramid
while True:
    height = get_int('Height: ')
    if height >= 1 and height <= 8:
        break

# Prints out left aligned pyramid line by line
for i in range(height):
    for k in range(height - i - 1):
        print(' ', end='')
    for j in range(i + 1):
        print('#', end='')
    print('')

